Hash incidents fix (#9653)
demisto/content: Packs/ML/ReleaseNotes/1_2_1.md

#### Scripts
##### HashIncidentsFields

> - Bug fix: do not return custom fields by deafult as part of the incident object

Reviewer: ```suggestion
- Fixed an issue where custom fields were returned by default as part of the incident object.
```

Once you update the docker image ping me again.
